util.go

```diff
@@ -10,6 +10,16 @@ import (
 	"strings"
 
 	"time"
+
+	pkgerrors "github.com/pkg/errors"
```

Comment:
What is the thinking behind doing this? personally I think keeping aliases out makes code reading clearer because then more people can rely on names of packages they already know.

Reply:
It's transitional, because some of the code stilluses "errors" and some of the code uses "github.com/pkg/errors". this lets us move stuff over slowly, and then we'll remove the package alias when everything's merged.

Reply:
:+1: makes total sense.